components/MovieSearchDisplay.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styles from '../styles/movieSearch.css';
+import { StyleSheet, css } from 'aphrodite';
+
+const MovieSearchDisplay = ({ movieSearchData }) => {
+    function getImageFromUrl(url) {
+        if (!url) return;
+
+        return `https://image.tmdb.org/t/p/w500${url}`;
+    }
+    function getGenresFromGenreList(genreList) {
+        // The genre list has all possible genres, but only the first few will
+        // be defined. We take these ids and return back what the actual genres are.
+        const movieGenres = [];
+        const genreHash = {
+            28: 'Action',
+            12: 'Adventure',
+            16: 'Animation',
+            35: 'Comedy',
+            80: 'Crime',
+            99: 'Documentary',
+            18: 'Drama',
+            10751: 'Family',
+            14: 'Fantasy',
+            36: 'History',
+            27: 'Horror',
+            10402: 'Music',
+            9648: 'Mystery',
+            10749: 'Romance',
+            878: 'Science Fiction',
+            10770: 'TV Movie',
+            53: 'Thriller',
+            10752: 'War',
+            37: 'Western',
+        };
+
+        genreList.forEach(genre => {
+            if (!genre) return;
+            
+            if (genreHash[genre]) {
+                movieGenres.push(genreHash[genre]);
+            }
+        });
+
+        return movieGenres;
+    }
+    function getReleaseYear(releaseData) {
+        // The release year comes in a string of format YYYY-MM-DD
+        // We only want the year, so just grab the first piece of data delimited by '-'
+        return releaseData.split('-')[0];
+    }
+    function getMovieGroupsToRender() {
```

Comment:
Groups movies in 4 and renders them within 1 group element. If there's less than 4 elements in the group when it's time to render, add invisible elements to make flexbox play nicely.